srcpkgs/goproxy/template

```diff
@@ -11,3 +11,4 @@ license="GPL-3.0-or-later"
 homepage="https://github.com/snail007/goproxy"
 distfiles="https://github.com/snail007/goproxy/archive/v${version}.tar.gz"
 checksum=13f8e37d274d2b7b7cdf1ae3d98723fbb9acf1476c92a1e0d43039109d238ccc
+broken="7.0 tag was removed from repository"
```

Comment:
Reviewing the upstream licensing this whole project appears kind of shady.  There is a v10.1 tag, is there a reason you did not use that instead of upgrading?

Reply:
By upgrading do you mean marking as broken?

I didn't want to update the package without having an idea of how it works / is supposed to work, and using a newer tag doesn't protect us from them dropping it in the future again. I'd consider dropping the package, if you think it makes sense.